### PR TITLE
Add final paper

### PR DESCRIPTION
**Pull request by boyinggong**

### CODE REVIEW

**Comment by BenjaminHsieh** [comment]
Is there a reason why we have all tex combined?

**Comment by boyinggong** [comment]
Because there are always some errors with the figure when I split them into different tex files.

**Comment by briankleinqiu** [comment]
path error or sizing?

**Comment by boyinggong** [comment]
It is not error, but the figures don't appear on the right place. (appear on other pages)

File: paper/report_final.tex [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+    \caption{t values of the gain coefficients for subject 2}
+\end{figure}
+
+\begin{figure}[H]
+    \centering
+        \includegraphics[scale=0.4]{figures/Regression2/t_loss_standard_sub2.png}
+    \caption{t values of the loss coefficients for subject 2}
+\end{figure}
+
+We can see that signigicant areas for the gain coefficients and those of the
```
Comment (briankleinqiu) [inline paper/report_final.tex]: can you replace this paragraph with this (fix some typoes): 

+We can see that significant areas for the gain coefficients and those of the 
+loss coefficients are mostly the same. This suggests the opposite of what 
+most people believe (that increasing potential losses should affect the areas 
+of the brain that mediate negative emotions in decision-making). Potential 
+losses were represented by decreasing activity in the same areas that are 
+sensitive to potential gains.

**Comment by boyinggong** [comment]
OK. I am still revising it. Please double check if there were any other typos, thanks.

File: paper/report_final.tex [hunk 2]
```diff
@@ -0,0 +1,563 @@
...
+
+\par 
+We determine the beta coefficients for each voxel in the subject's brain based on a linear regression on the BOLD signal data. We use these coefficients as evidence for increasing activation in certain voxels in the brain when the subject is making a decision to accept or reject, and so in this way we have a proxy for which regions of the brain are involved with loss-aversion decision-making. We also determine the p-values to validate these results.
+
+\par 
+We also run a logistic regression to determine each subject's willingness to accept or reject based on the values of potential loss and gain. This is the measure of behavioral loss aversion, and in the end we correlate this with ou neural loss aversion, making it clear that increasing gains correspond to increasing activity in certain parts of the brain.  
+
+\par 
+We also aim to investigate briefly if there are any statistically significant variations between the male and female subjects in addition to showing the brain activations through various graphs.
```
Comment (brianqiu) [inline paper/report_final.tex]: delete this part

File: paper/report_final.tex [hunk 3]
```diff
@@ -0,0 +1,563 @@
...
+
+\par 
+We determine the beta coefficients for each voxel in the subject's brain based on a linear regression on the BOLD signal data. We use these coefficients as evidence for increasing activation in certain voxels in the brain when the subject is making a decision to accept or reject, and so in this way we have a proxy for which regions of the brain are involved with loss-aversion decision-making. We also determine the p-values to validate these results.
+
+\par 
+We also run a logistic regression to determine each subject's willingness to accept or reject based on the values of potential loss and gain. This is the measure of behavioral loss aversion, and in the end we correlate this with ou neural loss aversion, making it clear that increasing gains correspond to increasing activity in certain parts of the brain.  
+
+\par
```
Comment (briankleinqiu) [inline paper/report_final.tex]: delete this paragraph about male/female

File: paper/report_final.tex [hunk 4]
```diff
@@ -0,0 +1,563 @@
...
+regression model.\\
+
+\begin{itemize}
+\item{QQ plot}
+\item{Residual - fitted plot}
+\end{itemize}
+
+
+\subsubsection{Multiple Test correction}
+
```
Comment (briankleinqiu) [inline paper/report_final.tex:259]: we're adding to the ROC, multiple test, cluster analysis, and anova subsubsections right?

File: paper/report_final.tex [hunk 5]
```diff
@@ -0,0 +1,563 @@
...
+\subsubsection{Inferences on data}
+
+After fitting regression models on our BOLD and behavioral data, we would try 
+assessing and validating our models. In order to do this, we would calculate 
+for the residual sum of squares for our model. We have to do three tests for 
+the model. The first one is that we calculate the t-statistics and p-value for 
+our beta coefficients to check whether our beta parameters are statistically 
+significant at a significance level of 5\%. The second one is that we calculate
+the residuals of this linear model and check whether it follows a normal 
+distribution. The third one is that we calculate the R-Squared value and the
```
Comment (briankleinqiu) [inline paper/report_final.tex]: Are we still doing the r-squared in paper

File: paper/report_final.tex (hunk 4 above)
Comment (boyinggong) [inline paper/report_final.tex:259]: Yes, but haven't finished yet.

File: paper/report_final.tex (hunk 5 above)
Comment (boyinggong) [inline paper/report_final.tex]: WE did calculated the r-squared but did not presented in the paper.

Comment (briankleinqiu) [inline paper/report_final.tex]: ok let's either add a brief mention that the r-squareds looked ok, or delete the sentence about r-squareds

**Pull request closed without merging**